docs/installing.md

```diff
@@ -51,16 +51,16 @@ Tracetest currently supports two traces backend: Jaeger and Grafana Tempo.
 
 Tracetest uses [Jaeger Query Service `16685` port](https://www.jaegertracing.io/docs/1.32/deployment/#query-service--ui) to find Traces using gRPC protocol.
 
-The commands below will install Tracetest connecting to the Jaeger tracing backend on `jaeger-query:16685`.
+The commands below will install Tracetest connecting to the Jaeger tracing backend on `jaeger-query.observability.svc.cluster.local:16685`.
 
 ```sh
 # Install Kubeshop Helm repo and update it
 helm repo add kubeshop https://kubeshop.github.io/helm-charts
 helm repo update
 
 helm install tracetest kubeshop/tracetest \
-  --set telemetry.dataStores.jaeger.jaeger.endpoint="jaeger-query:16685" \ # update this value to point to your jaeger install
-  --set telemetry.exporters.collector.exporter.collector.endpoint="otel-collector:4317" \ # update this value to point to your collector install
+  --set telemetry.dataStores.jaeger.jaeger.endpoint="jaeger-query.observability.svc.cluster.local:16685" `# update this value to point to your jaeger` install` \
```

Comment:
Ah, great shout on the extra tick. Definitely need to remove that.

But to also answer your question, the reason for the ticks is to actively "execute" that part of the code (which is a no-op since it is a comment) rather than have it read as an extra parameter to the helm command.

I was wondering if this is just a mac thing, but same issue when running on Ubuntu as well. As I said in the updated comment though, my need to even run this code was caused by own mistaken reading. So if you think this is extra confusing for folks don't stress too much on it!

For more context...

Here is a screenshot of the error on my mac:
![image](https://user-images.githubusercontent.com/1557346/188178192-04ab0cbc-644f-4b3d-a812-b848ad93614f.png)

and output from running on ubuntu:
```
root@54165761a813:/# uname -a
Linux 54165761a813 5.10.104-linuxkit #1 SMP PREEMPT Thu Mar 17 17:05:54 UTC 2022 aarch64 aarch64 aarch64 GNU/Linux
root@54165761a813:/# curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
root@54165761a813:/# chmod 700 get_helm.sh
root@54165761a813:/# ./get_helm.sh
Downloading https://get.helm.sh/helm-v3.9.4-linux-arm64.tar.gz
Verifying checksum... Done.
Preparing to install helm into /usr/local/bin
helm installed into /usr/local/bin/helm
root@54165761a813:/# helm repo add kubeshop https://kubeshop.github.io/helm-charts
"kubeshop" has been added to your repositories
root@54165761a813:/# helm repo update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "kubeshop" chart repository
Update Complete. ⎈Happy Helming!⎈
root@54165761a813:/# helm install tracetest kubeshop/tracetest \
  --set telemetry.dataStores.jaeger.jaeger.endpoint="jaeger-query:16685" \ # update this value to point to your jaeger install
  --set telemetry.exporters.collector.exporter.collector.endpoint="otel-collector:4317" \ # update this value to point to your collector install
Error: INSTALLATION FAILED: expected at most two arguments, unexpected arguments:  #, update, this, value, to, point, to, your, jaeger, install
bash: --set: command not found
```